Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "RFFT"
 uuid = "3bd9afcd-55df-531a-9b34-dc642dce7b95"
 authors = ["Tim Holy <tim.holy@gmail.com>"]
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
Unless you expect more breaking changes soon, I suspect we should just go for 1.0.